central/detection/service/service_impl.go

```diff
@@ -141,6 +141,7 @@ func (s *serviceImpl) DetectBuildTime(ctx context.Context, req *apiV1.BuildDetec
 		return nil, err
 	}
 	enrichmentContext.FetchOpt = fetchOpt
+	enrichmentContext.Delegable = true
```

Comment:
when is this ever false? in another PR?

Reply:
It defaults to false, so all the workflows that are not explicitly setting it, it will be false (reprocessing for example)

Reply:
I just wasn't sure if there was a spot where it wasn't set, thanks